tests/unit/CountItems/CountWorker.js

```diff
@@ -22,7 +21,6 @@ describe('CountItems::CountWorker', () => {
                     setup: [null],
                     close: [],
                     isConnected: false,
-                    getIsTransient: [],
```

Comment:
in the previous code, we used to have different `getIsTransient()` mock results depending on the test case : i.e. either `[]` (i.e. no error, not transient) or `[null, true]` (i.e. no error, transient)

However, with your change the mock always returns `true` : and we don't actually test the "usual" scenario we are in...